src/components/footer/button-link.vue

```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+const props = defineProps<{
+  label: string
+  link: string
+}>()
+</script>
+
+<template>
+  <span
+    class="h-full flex flex-col justify-center px-[7px] uppercase hover:text-white hover:bg-primary"
+  >
+    <a :href="`${props.link}`" target="_blank">{{ $props.label }}</a>
```

Comment:
I think only <a> tag should be used, this way the link could take the entire width/height (like in prod) and it would be easier to clicked on it.

Reply:
Good call, thanks!